### PR TITLE
Followups to the ImageDescriptor pull request.

### PR DESCRIPTION
**Pull request by nical**
r? @kvark

---
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/762)

### CODE REVIEW

File: wrench/src/json_frame_writer.rs [hunk 1]
```diff
@@ -204,16 +204,9 @@ impl webrender::ApiRecordingReceiver for JsonFrameWriter {
...
-                    stride
-                } else {
-                    match descriptor.format {
-                        ImageFormat::A8 => descriptor.width,
-                        ImageFormat::RGBA8 | ImageFormat::RGB8 => descriptor.width*4,
-                        ImageFormat::RGBAF32 => descriptor.width*16,
-                        _ => panic!("Invalid image format"),
-                    }
-                };
+                let stride = descriptor.stride.unwrap_or_else(||{
```
Comment (kvark) [inline wrench/src/json_frame_writer.rs]: These could just be `unwrap_or`, but I will not lose my sleep over it ;)

**Comment by kvark** [comment]
Thanks!
@bors-servo r+

File: wrench/src/json_frame_writer.rs (hunk 1 above)
Comment (nical) [inline wrench/src/json_frame_writer.rs]: ah yes indeed

**Pull request merged**